package.json

```diff
@@ -1,12 +1,13 @@
 {
-  "license": "GPLv2",
+  "license": "GPL-2.0-or-later",
```

Comment:
Good catch!

Reply:
Yeah, your initial thought was correct and yarn will complain otherwise 😄